### PR TITLE
Updated dependencies and compilied for cuba 6.10

### PR DESCRIPTION
**Pull request by utrost**
Used MySQL as Database, updated dependencies for cuba 6.10

### CODE REVIEW

**Comment by mariodavid** [comment]
hi,

thanks for contribution! Really appreciate it...

Will look into the changes and give you feedback before we merge it.

File: build.gradle [hunk 1]
```diff
@@ -89,12 +89,12 @@ subprojects {
...
     appComponent("com.haulmont.cuba:cuba-global:$cubaVersion")
-    appComponent("de.balvi.cuba.declarativecontrollers:declarativecontrollers-global:0.6.0")
-    appComponent("de.diedavids.cuba.entitysoftreference:entity-soft-reference-global:0.3.2")
+    appComponent("de.balvi.cuba.declarativecontrollers:declarativecontrollers-global:0.7.0")
+    appComponent("de.diedavids.cuba.entitysoftreference:entity-soft-reference-global:0.4.0")
 
 }
 
-def hsql = 'org.hsqldb:hsqldb:2.2.9'
+def mysql = 'mysql:mysql-connector-java:5.1.46'
```
Comment (mariodavid) [inline build.gradle]: can you switch back to HSQLDB for later development? Doing it via studio does not require you to change all the code snippets. thanks!

File: modules/core/db/update/mysql/18/181229-0-dropCustomer.sql [hunk 2]
```diff
@@ -0,0 +1,2 @@
+rename table ddct_customer to DDCT_CUSTOMER__U47525 ;
```
Comment (mariodavid) [inline modules/core/db/update/mysql/18/181229-0-dropCustomer.sql]: you can remove those. Those are internal test classes - that should not be generated. Can be deleted.

File: modules/core/db/update/mysql/18/181229-0-dropCustomer_DropScript.sql [hunk 3]
```diff
@@ -0,0 +1 @@
+drop table if exists DDCT_CUSTOMER__U47525 cascade ;
```
Comment (mariodavid) [inline modules/core/db/update/mysql/18/181229-0-dropCustomer_DropScript.sql]: you can remove those. Those are internal test classes - that should not be generated. File can be deleted.

File: modules/core/db/update/mysql/18/181229-0-dropTagTest.sql [hunk 4]
```diff
@@ -0,0 +1 @@
+rename table ddct_tag_test to DDCT_TAG_TEST__U44381 ;
```
Comment (mariodavid) [inline modules/core/db/update/mysql/18/181229-0-dropTagTest.sql]: you can remove those. Those are internal test classes - that should not be generated. File can be deleted.

File: modules/core/db/update/mysql/18/181229-1-createCustomer.sql [hunk 5]
```diff
@@ -0,0 +1,14 @@
+create table DDCT_CUSTOMER (
```
Comment (mariodavid) [inline modules/core/db/update/mysql/18/181229-1-createCustomer.sql]: you can remove those. Those are internal test classes - that should not be generated. File can be deleted.

File: modules/core/db/update/mysql/18/181229-0-dropTagTest_DropScript.sql [hunk 6]
```diff
@@ -0,0 +1 @@
+drop table if exists DDCT_TAG_TEST__U44381 cascade ;
```
Comment (mariodavid) [inline modules/core/db/update/mysql/18/181229-0-dropTagTest_DropScript.sql]: you can remove those. Those are internal test classes - that should not be generated. File can be deleted.

File: modules/core/db/update/mysql/18/181229-1-createTagTest.sql [hunk 7]
```diff
@@ -0,0 +1,14 @@
+create table DDCT_TAG_TEST (
```
Comment (mariodavid) [inline modules/core/db/update/mysql/18/181229-1-createTagTest.sql]: Seems to be your internal test class - right? should be deleted before merging

File: modules/core/db/update/mysql/18/181229-2-updateTagging.sql [hunk 8]
```diff
@@ -0,0 +1,2 @@
+alter table DDCT_TAGGING add column CUSTOMER_ID varchar(32) ;
```
Comment (mariodavid) [inline modules/core/db/update/mysql/18/181229-2-updateTagging.sql]: you can remove those. Those are internal test classes - that should not be generated. File can be deleted.

File: modules/core/db/update/mysql/18/181229-2-updateTagging01.sql [hunk 9]
```diff
@@ -0,0 +1,2 @@
+alter table DDCT_TAGGING add constraint FK_DDCT_TAGGING_ON_CUSTOMER foreign key (CUSTOMER_ID) references DDCT_CUSTOMER(ID);
```
Comment (mariodavid) [inline modules/core/db/update/mysql/18/181229-2-updateTagging01.sql]: you can remove those. Those are internal test classes - that should not be generated. File can be deleted.

File: modules/core/db/update/mysql/18/181229-2-updateTagging02.sql [hunk 10]
```diff
@@ -0,0 +1 @@
+update DDCT_TAGGING set DTYPE = 'ddct$Tagging' where DTYPE is null ;
```
Comment (mariodavid) [inline modules/core/db/update/mysql/18/181229-2-updateTagging02.sql]: you can remove those. Those are internal test classes - that should not be generated. File can be deleted.

File: modules/core/db/update/mysql/18/181229-2-updateTagging03.sql [hunk 11]
```diff
@@ -0,0 +1,3 @@
+alter table DDCT_TAGGING change column DTYPE DTYPE__U80164 varchar(100)^
```
Comment (mariodavid) [inline modules/core/db/update/mysql/18/181229-2-updateTagging03.sql]: you can remove those. Those are internal test classes - that should not be generated. File can be deleted.

File: modules/core/db/update/mysql/18/181229-2-updateTagging03_DropScript.sql [hunk 12]
```diff
@@ -0,0 +1,2 @@
+alter table DDCT_TAGGING drop column CUSTOMER_ID__U15937 cascade ;
```
Comment (mariodavid) [inline modules/core/db/update/mysql/18/181229-2-updateTagging03_DropScript.sql]: you can remove those. Those are internal test classes - that should not be generated. File can be deleted.

File: modules/web/src/de/diedavids/cuba/taggable/web/messages.properties [hunk 13]
```diff
@@ -11,4 +11,5 @@ actions.Ok=Ok
 actions.Cancel=Cancel
 
 actions.tags=
-column.tags=Tags
\ No newline at end of file
+column.tags=Tags
+menu-config.ddct$TagTest.browse = Tag Tests
```
Comment (mariodavid) [inline modules/web/src/de/diedavids/cuba/taggable/web/messages.properties]: Seems to be your internal test class - right? should be deleted before merging

**Review by mariodavid — CHANGES_REQUESTED** [review]
looks great! please remove not needed files, push again and then we merge & release...

**Comment by utrost** [comment]
Removed Test data, set HSQLDB as default again.

**Review by mariodavid — APPROVED** [review]

**Comment by mariodavid** [comment]
thanks for your contribution.

I'll merge it and release 0.3.0 with 6.10 support and MySQL support as well.

If you want you can continue with the attachable addon.

Bye

**Pull request merged**